src/utilities/global.ts

```diff
@@ -51,7 +56,7 @@ const escapeDoubleQuotes = (s: string) => {
 
 export function escapeQuotes(s: string): string {
   if (s.includes("'") && s.includes('"')) {
-    return `%q(${removeNewlinePlaceholders(s)})` 
+    return `%q(${escapeSpecialCaseBackslashes(removeNewlinePlaceholders(s))})`
```

Comment:
these special case backslashes are only an issue with %q strings, not ' or "?

Reply:
yes